src/FSharpLint.Core/Rules/Conventions/FavourStaticEmptyFields.fs

```diff
@@ -0,0 +1,85 @@
+module FSharpLint.Rules.FavourStaticEmptyFields
+
+open FSharpLint.Framework
+open FSharpLint.Framework.Suggestion
+open FSharp.Compiler.Syntax
+open FSharp.Compiler.Text
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.Rules
+open System
+open System.Collections.Generic
+
+let private getStaticEmptyErrorMessage  (range:FSharp.Compiler.Text.Range) (idText:string) =
+    let formatError errorName =
+        String.Format(Resources.GetString errorName, idText)
+
+    "RulesFavourStaticEmptyFieldsError" |> formatError |> Array.singleton
+
+let private generateError (range:FSharp.Compiler.Text.Range) (idText:string) =
+    if idText = "" || idText = "[]" then
```

Comment:
please also include `[| |]`, which should be caught by the rule and suggest Array.empty instead

Reply:
@janus you didn't address this yet

Reply:
It is already acted upon. Sorry that I didn't flag resolved. Check test and code base if in doubt.

Reply:
how about `[ ]`? (with a space in the middle); I don't see a test for that

Reply:
This state would be seen ass [](which is already there), however let me add it in the test.